web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -1344,5 +1344,8 @@
     "ui-editorIndentType-colored": "Colored indents",
     "filterSettings": "Filter settings ",
     "filterPlaceHolder": "e.g. INSPIRE",
-    "filterReset": "Reset the filter"
```

Comment:
Why is this string removed?

Reply:
Thanks for checking, was removed accidentally. Going to re-add it.